public/data/index.ts

```diff
@@ -1,22 +1,95 @@
-import { StatProps } from '../../src/utils/interfaces/props'
+import { StatProps, IconCardProps, InfoSectionProps } from "../../src/utils/interfaces/props";
+import {
+  faBoxesStacked,
+  faChartSimple,
+  faLandmark,
+  faPeopleGroup,
+} from "@fortawesome/free-solid-svg-icons";
 
 export const stats: StatProps[] = [
+  {
+    stat: "4",
+    statName: "Projects",
+    subtitle: "(Per Semester)",
+  },
+  {
+    stat: "30+",
+    statName: "Majors",
+  },
+  {
+    stat: "200+",
+    statName: "Members",
+    subtitle: "And Alumni",
+  },
+  {
+    stat: "1",
+    statName: "DeCal",
+  },
+];
+
+export const aboutPageIconCards: IconCardProps[] = [
+  {
+    title: "Be Interdisciplinary",
+    children: `We offer a wide range of projects, general 
```

Comment:
Not to be pedantic, but is there a reason we call it 'children' rather than 'description;?

Reply:
Yes, calling a prop `children` allows us to pass in the information directly between the resulting HTML tags. I.e instead of 
```
<IconCard title={title} body={body}></IconCard>
```
We can just do
```
<IconCard title={title}>{body}</IconCard>
```

I personally think its a bit cleaner to assign `children` as the main part of the element. Also since we have the interface and a JSDocs comment I think its not a big deal.